packages/data-broker/src/__tests__/services/dhis/DhisService.stubs.js

```diff
@@ -34,12 +35,17 @@ export const stubDhisApi = ({
       .mockImplementation(async codes =>
         codes.map(code => ({ code, id: DATA_ELEMENTS[code].uid, valueType: 'NUMBER' }), {}),
       ),
-    getServerName: jest.fn().mockReturnValue(SERVER_NAME),
     getResourceTypes: jest.fn().mockReturnValue({ DATA_ELEMENT: 'dataElement' }),
+    serverName,
   });
-  jest.spyOn(GetDhisApiInstance, 'getDhisApiInstance').mockReturnValue(dhisApi);
+};
 
-  return dhisApi;
+export const stubGetDhisApi = mockDhisApi => {
```

Comment:
Split this into two functions because we don't always need to stub as well as mock this import.